Zend/tests/return_types/026.phpt

```diff
@@ -0,0 +1,9 @@
+--TEST--
+Return type of parent is not allowed in function
+--FILE--
+<?php
+
+function test(): parent {}
+
+--EXPECTF--
+Fatal error: Cannot declare a return type of parent outside of a class scope in %s on line 3
```

Comment:
Just to make sure I didn't miss anything: These two tests are unrelated to your changes, right?